### PR TITLE
Update docker readme for M1 users [#8421]

### PR DESCRIPTION
**Pull request by matisnape**
#### What? Why?

Relates to [#8421]

I'm a M1 Pro mac user and I wanted to setup the app via Docker, as it is the least troublesome way, since this architecture is fairly new and not supported by a lot of libraries. 
However, this project has a Dockerfile that doesn't work with M1 out of the box and since almost no one in the dev team uses Docker for the project (at least that's [what I've learned on Slack](https://openfoodnetwork.slack.com/archives/C2GQ45KNU/p1665262863935319)), there is little incentive for updating it.

So after some digging I've prepared some workaround steps that may help other newcomers ;)

I haven't managed to fix the Chrome & Chromedriver issue, but I figure these are for running tests so nothing pressing yet.
I've removed `[arch=amd64]` from the Chrome command, but I'm still getting 
```
#13 3.095 E: Unable to locate package google-chrome-stable
------
failed to solve: executor failed running [/bin/sh -c wget --quiet -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - &&     sh -c "echo 'deb http://dl.google.com/linux/chrome/deb/ stable main' >> /etc/apt/sources.list.d/google-chrome.list" &&     apt-get update &&     apt-get install -fy google-chrome-stable]: exit code: 100
```
So [this solution](https://stackoverflow.com/questions/27423684/unable-to-locate-package-google-chrome-stable-ubuntu12-on-openstack) didn't help. Maybe there's someone I could reach out to make this work?  

Oh, and there's another thing - solving one error requires bumping `mini-racer` from 0.4.0 to 0.5.0 - what do you think about making this change permanent in the Gemfile?

#### Release notes

Changelog Category: Documentation changes

PS. I would be grateful if this PR could be accepted as Hacktoberfest contribution :)

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -68,6 +68,30 @@ Just change the entry in ~/.docker/config.json like this (credStore instead of c
...
+```
+#0 3.023 E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/bionic-security/main/binary-arm64/Packages  404  Not Found [IP: 185.125.190.36 80]
+#0 3.023 E: Some index files failed to download. They have been ignored, or old ones used instead.
+------
+failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y   curl   git   build-essential   software-properties-common   wget   zlib1g-dev   libreadline-dev   libyaml-dev   libffi-dev   libxml2-dev   libxslt1-dev   wait-for-it   imagemagick   unzip   libjemalloc-dev   libssl-dev   ca-certificates   gnupg]: exit code: 100
+```
+To solve this, we need to hack Dockerfile a bit.
+
+Steps to follow:
+- Comment out line 7 `RUN echo "deb http://security.ubuntu.com/ubuntu bionic-security main" >> /etc/apt/sources.list`
```
Comment (mkllnk) [inline docker/README.md:83]: I'm wondering if we can remove this line from the file. Commit 4a60d7ef0ee4ea3e6b939f27b0587f7079f778d7 introduced it but it doesn't say why it needs the repo.

File: docker/README.md [hunk 2]
```diff
@@ -68,6 +68,30 @@ Just change the entry in ~/.docker/config.json like this (credStore instead of c
...
+- Comment out line 33 (`ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so`) and remove `RUBY_CONFIGURE_OPTS=--with-jemalloc` from the start of line 42
```
Comment (mkllnk) [inline docker/README.md:84]: #7717 introduced this but I think we can take it out again. It's not general enough to work on all platforms. On our servers we install the libjemalloc-dev apt package and we don't need this environment variable. We could also try that in Docker.

File: docker/README.md [hunk 3]
```diff
@@ -68,6 +68,30 @@ Just change the entry in ~/.docker/config.json like this (credStore instead of c
...
+
+You may also need to comment out stuff related to Chromedriver and Chrome. Chrome setup may work with `[arch=amd64]` removed.
+
+See [#8421](https://github.com/openfoodfoundation/openfoodnetwork/issues/8421) for more info
+
+2. `stack smashing detected ***: terminated`
+To fix this error on server startup, you need to bump `mini-racer` gem from `0.4.0` to `0.5.0`
```
Comment (mkllnk) [inline docker/README.md:91]: Yes, let's try that in another PR for everyone.

**Review by mkllnk — APPROVED** [review]
Thank you!

I think that it's worth trying to solve these issues in code for everyone. Some ideas below.

And with Chrome, maybe we can just use apt to install the Ubuntu system's version instead of an additional repository which will most likely fail from time to time.

So if you are happy to continue on this, I would merge these doc updates and then you can remove parts step by step if you solve these issues in general. How does that sound?

**Review by jibees — APPROVED** [review]
thanks for your contribution!

Still waiting for the answers to @mkllnk questions ;)

**Pull request merged**